server/Dockerfile

```diff
@@ -4,14 +4,26 @@ FROM jboss/base-jdk:8
 # Set the INFINISPAN_SERVER_HOME env variable
 ENV INFINISPAN_SERVER_HOME /opt/jboss/infinispan-server
 
+ENV INFINISPAN_SHA a55ca9cddd5b3373029b780f6530ccdffdc8396f
+
 # Set the INFINISPAN_VERSION env variable
-ENV INFINISPAN_VERSION 8.2.0.Final
+ENV INFINISPAN_VERSION 9.0.0.Alpha3
 
-# Download and unzip Infinispan server
-RUN cd $HOME && curl http://downloads.jboss.org/infinispan/$INFINISPAN_VERSION/infinispan-server-$INFINISPAN_VERSION-bin.zip | bsdtar -xf - && mv $HOME/infinispan-server-$INFINISPAN_VERSION $HOME/infinispan-server && chmod +x /opt/jboss/infinispan-server/bin/standalone.sh
+ENV MGMT_USER admin
 
-# Expose Infinispan server  ports 
-EXPOSE 8080 8181 9990 11211 11222 
+ENV MGMT_PASS admin
+
+# Server download location
+ENV DISTRIBUTION_URL https://repo1.maven.org/maven2/org/infinispan/server/infinispan-server-build/$INFINISPAN_VERSION/infinispan-server-build-$INFINISPAN_VERSION.zip
```

Comment:
I used maven central because for me it's 100x faster to download than Nexus. Is that the same for you?